modules/logging/ui/src/sap/commerce/toolset/logging/ui/CxCustomLogTemplateDialog.kt

```diff
@@ -64,11 +67,59 @@ class CxCustomLogTemplateDialog(
                 }
             )
                 .label("Default level:")
-                .bindItem(mutable.defaultEffectiveLevel)
+                .bindItem(context.mutable.defaultEffectiveLevel)
                 .align(AlignX.FILL)
         }.layout(RowLayout.PARENT_GRID)
+
+        afterPanel()
+    }
+
+    open fun Panel.beforePanel() {
+
+    }
+
+    open fun Panel.afterPanel() {
+
     }
 
     override fun getPreferredFocusedComponent(): JComponent = nameTextField
 
+}
+
+class CxAdvancedCustomLogTemplateDialog(
+    context: LogTemplateDialogContext
+) : CxCustomLogTemplateDialog(context) {
```

Comment:
the same can be achieved with the `CxCustomLogTemplateDialog` and new flag in the `LogTemplateDialogContext` which will be responsible for render/apply options.